modules/doc/content/application_usage/tracked_apps.md

```diff
@@ -1,44 +1,137 @@
+
+
+### Mechanics
+
+- [Blackbear](https://github.com/idaholab/blackbear) - Degradation processes in concrete and other structural materials
+- [Deer](https://github.com/Argonne-National-Laboratory/deer) - Various mechanics modules for MOOSE
+- [MASTODON](https://github.com/idaholab/mastodon) - Structural dynamics, seismic analysis, and risk assessment
+- [RACCOON](https://github.com/hugary1995/raccoon) - Massively parallel FEM code for phase-field for fracture
+- [Okami](https://hpcgitlab.hpc.inl.gov/idaholab/okami/) - Mixed Oxide fuel studies
+
+### Materials
+
+- [GRIME](https://github.com/shortlab/grime) - The Grand Radiation Informed Microstructural Evolver: Coupled, spatially dependent radiation damage and microstructural evolution
 - [HOGNOSE](https://github.com/shortlab/hognose) - CASL's mesoscale corrosion/oxidation code
-- [MAMBA](https://github.com/shortlab/mamba) - CASL's code for boiling, heat transfer, and chemistry in porous media
 - [Hyrax](https://github.com/UMThorntonGroup/Hyrax) - Zirconium hydride precipitation and growth in LWR nuclear fuel cladding
+- [Magpie](https://github.com/idaholab/magpie) - Mesoscale Atomistic Glue Program for Integrated Execution
+- [Pika](https://github.com/idaholab/pika) - Phase-field model for micro-structure evolution of ice
+
+
+### Electromagnetics, Plasmas and Additive Manufacturing
+
+- [Apollo](https://github.com/aurora-multiphysics/apollo) - Enabling 3D electromagnetics simulation in MOOSE, using the MFEM FE library.
+- [CRANE](https://crane-plasma-chemistry.readthedocs.io/en/latest/) - Chemical ReAction NEtworks for plasma chemistry and thermochemistry problems.
 - [Ferret](https://mangerij.github.io/ferret/) - Kernels to implement the time-dependent Landau-Ginzburg theory of phase transitions for simulating ferroelectric materials
-- [GRIME](https://github.com/shortlab/grime) - The Grand Radiation Informed Microstructural Evolver: Coupled, spatially dependent radiation damage and microstructural evolution
-- [Gardensnake](https://github.com/friedmud/gardensnake) - Nodal neutron diffusion code developed at MIT
+- [MALAMUTE](https://mooseframework.inl.gov/malamute/) - Advanced manufacturing modeling and simulation
+- [Orpheus](https://github.com/aurora-multiphysics/orpheus) - Additional electromagnetics solvers in MOOSE
+- [phaethon](https://github.com/aurora-multiphysics/phaethon) - Fast ion heat fluxes produced by ASCOT5 in MOOSE multiphysics simulations
 - [Zapdos](https://github.com/shannon-lab/zapdos) - Low temperature plasma simulation
-- [Moltres](https://github.com/arfc/moltres) - Molten salt reactor simulation
+
+
+### Chemistry & species transport
+
 - [DGOSPREY](https://github.com/aladshaw3/dgosprey) - Discontinuous Galerkin Off-gas SeParation and REcoverY model: joint development between INL and GIT
+- [MAMBA](https://github.com/shortlab/mamba) - CASL's code for boiling, heat transfer, and chemistry in porous media
```

Comment:
I would move this to "Legacy", and add something like "MOOSE port of the still maintained non-MOOSE MAMBA CASL code" or something like that.